module/zfs/dmu.c

```diff
@@ -2168,6 +2170,12 @@ dmu_write_policy(objset_t *os, dnode_t *dn, int level, int wp, zio_prop_t *zp)
 		compress = zio_compress_select(os->os_spa, dn->dn_compress,
 		    compress);
 
+#if 0
+/* XXX: Allan: Does the dnode need the compress level? */
```

Comment:
I think this should be able to get answered/tested before merging...

Reply:
@BrainSlayer Could you answer or delete this? 
In-code commenting is not the place for review.